oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -489,11 +490,13 @@ private Map<String, String> buildDimensions(
                         .getCredentialTrustLevel()
                         .equals(CredentialTrustLevel.LOW_LEVEL);
         var levelOfConfidence = LevelOfConfidence.NONE.getValue();
+        // Assumption: Requested vectors of trust will either all be for identity or none, and so we
+        // can check just the first
         if (orderedVtrList.get(0).containsLevelOfConfidence()) {
-            levelOfConfidence = orderedVtrList.get(0).getLevelOfConfidence().getValue();
+            levelOfConfidence = VectorOfTrust.stringifyLevelsOfConfidence(orderedVtrList);
         }
         dimensions.put("MfaRequired", mfaRequired ? "Yes" : "No");
-        dimensions.put("RequestedLevelOfConfidence", levelOfConfidence);
+        dimensions.put("RequestedLevelOfConfidence", format("[%s]", levelOfConfidence));
```

Comment:
Normally I'd support wrapping this in square brackets, but as is we've got loads of legacy data with a single string this will be incompatible with. All the `P2` will now be `[P2]` and the graphs will break, right?